src/index.d.ts

```diff
@@ -0,0 +1,326 @@
+    card?: boolean;
+    center?: boolean;
+    flex?: boolean | number;
+    fluid?: boolean;
+    height?: number;
+    left?: boolean;
+    middle?: boolean;
+    right?: boolean;
+    row?: boolean;
+    safe?: boolean;
+    shadow?: boolean;
+    shadowColor?: boolean;
+    space?: string;
+    top?: boolean;
+    width?: number;
+  }
+  export class Block extends React.Component<BlockProps> {}
+
+  export interface ButtonProps extends TouchableOpacityProps, BaseProps {
+    capitalize?: boolean;
+    color?: ButtonColorType;
+    disabled?: boolean;
+    icon?: string;
+    iconColor?: boolean | string;
+    iconFamily?: boolean | string;
+    iconSize?: number;
+    loading?: boolean;
+    loadingSize?: 'small' | 'large';
+    lowercase?: boolean;
+    onlyIcon?: boolean;
+    opacity?: number;
+    radius?: number;
+    shadowColor?: boolean | string;
+    shadowless?: boolean;
+    size?: 'small' | 'large' | number;
+    uppercase?: boolean;
+  }
+  export class Button extends React.Component<ButtonProps> {}
+
+  export interface CardProps extends BaseProps {
+    card?: boolean;
+    shadow?: boolean;
+    borderless?: boolean;
+    image?: string;
+    imageBlockStyle?: strubg;
+    imageStyle?: ImageStyle;
+    avatar?: string;
+    location?: string;
+    locationColor?: boolean | string;
+    title?: string;
+    titleColor?: string;
+    caption?: string;
+    captionColor?: string;
+    footerStyle?: ViewStyle;
+  }
+  export class Card extends React.Component<CardProps> {}
+
+  export interface CheckBoxProps extends BaseProps {
+    checkboxStyle?: ViewStyle;
+    disabled?: boolean;
+    flexDirection?: ViewStyle['flexDirection'];
+    iconName?: string;
+    iconSize?: number;
+    iconFamily?: IconFamilyType;
+    image?: string;
+    imageStyle?: ImageStyle;
+    initialValue?: boolean;
+    label?: string;
+    labelStyle?: TextStyle;
+    onChange?: () => void;
+  }
+  export class CheckBox extends React.Component<CheckBoxProps> {}
+
+  export interface DeckSwiperProps extends BaseProps {
+    style?: ViewStyle;
+    components?: ReactNode[];
+    onSwipeRight?: () => void;
+    onSwipeLeft?: () => void;
+    focusedElementStyle?: ViewStyle;
+    nextElementStyle?: ViewStyle;
+  }
+  export class DeckSwiper extends React.Component<DeckSwiperProps> {}
+
+  export interface IconProps extends BaseProps {
+    name?: string;
+    family?: IconFamilyType;
+    size?: number;
+    color?: string;
+  }
+  export class Icon extends React.Component<IconProps> {}
+
+  export interface InputProps
+    extends Omit<
+        TextInputProps,
+        | 'style'
+        | 'keyboardType'
+        | 'secureTextEntry'
+        | 'placeholderTextColor'
+        | 'underlineColorAndroid'
+      >,
+      BaseProps {
+    type?: TextInputProps['keyboardType'];
+    password?: boolean;
+    label?: string;
+    bgColor?: string;
+    rounded?: boolean;
+    borderless?: boolean;
+    viewPass?: boolean;
+    icon?: string;
```

Comment:
Can you add the `iconContent` prop?
https://github.com/galio-org/galio/blob/master/src/Input.js#L152